### ISSUE TITLE
torch/flaslight availabilty to all devices

### ISSUE THREAD

**Issue by Fuseteam** [ubports/indicator-power#11]
currently the flashlight toggle is not available on all devices
there is, however an app that can serve the same function on many devices in the store "utorch"
can the implementation from utorch be used to make the toggle available on more devices?

**Comment by Fuseteam** [comment]
the flashlight [works with this](https://github.com/ubports/indicator-power/blob/xenial/src/flashlight.c#L33)

**Comment by dobey** [comment]
> can the implementation from utorch be used to make the toggle available on more devices?

No. uTorch is simply using the `Camera` component to toggle the flash on or off. It's not the correct way to do flashlight.

**Comment by Fuseteam** [comment]
ah and it seems some devices are simply missing the files to do flashlight
